### DISCUSSION TITLE
It seems that there are still some repositories that have not been migrated.

### DISCUSSION THREAD

**Discussion by daiaji** [zdharma-continuum/zinit#132] [Q&A] [answered]
Such as these repositories.
```
      psprint/zsh-navigation-tools \
      zinit-zsh/z-a-rust \
      zinit-zsh/z-a-as-monitor \
      zinit-zsh/z-a-patch-dl \
      zinit-zsh/z-a-bin-gem-node \
```

**Comment by pschmitt** [comment, marked as answer]
These have all been migrated already.

- https://github.com/zdharma-continuum/zsh-navigation-tools/
- https://github.com/zdharma-continuum/zinit-annex-rust/
- https://github.com/zdharma-continuum/zinit-annex-readurl
- https://github.com/zdharma-continuum/zinit-annex-patch-dl
- https://github.com/zdharma-continuum/zinit-annex-bin-gem-node
